dist/XSLoader/XSLoader_pm.PL

```diff
@@ -9,11 +11,13 @@ print OUT <<'EOT';
 # Generated from XSLoader_pm.PL (resolved %Config::Config value)
 # This file is unique for every OS
 
-package XSLoader;
+use strict;
+no strict 'refs';
+#use warnings;
```

Comment:
These warnings comments probably either need some explanation or should be omitted.

Reply:
It's meant for a follow on discussion. I'll pull it out for now if it's distracting 😄